app/controllers/volunteers_controller.rb

```diff
@@ -68,6 +74,6 @@ def remove
     def volunteer_params
       params.require(:volunteer).permit(:picture,:first_name,:last_name,:email,
       :mobile_phone, :birthdate, :gender, :city, :state, :employer, :position,
-      :availability, :skill, :volunteer_notes)
+      :availability, :skill, :volunteer_notes, :remove)
```

Comment:
I was trying to add a function I saw in a tutorial video so that the user can remove the image they uploaded. I couldn't figure out how to get it to work so I just removed it, I forgot to remove it from the controller. I removed the "remove" parameter.